### PR TITLE
New choices for forest story

### PR DESCRIPTION
**Pull request by keldavis**

### CODE REVIEW

**Comment by keldavis** [comment]
Please review this, @casey-collab

File: english/forest/drag/snake/slither/help.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+You slither towards towards the person, hissing for help. Turns out it
```
Comment (casey-collab) [inline english/forest/drag/snake/slither/help.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/forest/drag/snake/slither/safety.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+You slither down a nice, warm, tight, wet hole. You feel very safe here.
```
Comment (casey-collab) [inline english/forest/drag/snake/slither/safety.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/forest/drag/snake/slither/help.md (hunk 1 above)
Comment (keldavis) [inline english/forest/drag/snake/slither/help.md]: Please review this, @casey-collab

**Pull request merged**